src/app/wallets/send-on-chain.ts

```diff
@@ -182,8 +181,13 @@ const executePaymentViaIntraledger = async ({
   if (recipientWallet.id === senderWallet.id) return new SelfPaymentError()
 
   // TODO Usd use case
-  assert(recipientWallet.currency === WalletCurrency.Btc)
-  assert(senderWallet.currency === WalletCurrency.Btc)
+  if (
+    !(
+      recipientWallet.currency === WalletCurrency.Btc &&
+      senderWallet.currency === WalletCurrency.Btc
+    )
+  )
+    return new NotImplementedError("USD intraledger")
```

Comment:
we should probably have a bracket here. (when a `if` and the result are not on the same line, using bracket avoid merging issue. (ie: https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/)